src/screens/Profile/TravelToken/index.tsx

```diff
@@ -40,7 +40,7 @@ export default function TravelCard({navigation}: TravelCardScreenProps) {
         leftButton={{type: 'back'}}
       />
       <ScrollView style={styles.scrollView}>
-        <TravelTokenBox showIfThisDevice={true} />
+        <TravelTokenBox showIfThisDevice={true} alwaysShowErrors={true} />
```

Comment:
what is the purpose of alwaysShowErrors if it is always going to be true? 🤔

Reply:
The `TravelTokenBox` is used in other parts in the code (not touched by this PR) where the `alwaysShowErrors` is not added, and thus `false`.

Reply:
makes sense 👍  Just so many flags 😅

Reply:
@gorandalum Thanks for the explanation, of course, there will more to improve but as is now I think is fine, @Nisha1306 also pointed out good things here :) I do not have more to add than trying to test this solution deeply.